src/lib/components/FormLayout/README.mdx

```diff
@@ -402,6 +402,7 @@ This is a demo of all components supported by FormLayout.
             </FormLayoutCustomField>
             <CheckboxField
```

Comment:
In `/src/lib/components/FormLayout/README.mdx` it says:

> CheckboxField and Toggle always display their labels after inputs inside FormLayout. In other words, their labels never appear in a column with other fields' labels even when they have labelPosition set to before. This feature https://github.com/react-ui-org/react-ui/issues/232 once [CSS subgrid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Subgrid) is fully supported in [browsers that we support](https://github.com/react-ui-org/react-ui/blob/master/getting-started/browsers-and-devices).

I think this needs to be updated.

(sorry for wrong place, but GH does not support PR wide discussions)

Reply:
Fixed, thanks for pointing it out!